website/notifications/emails.py

```diff
@@ -87,7 +102,9 @@ def store_emails(recipient_ids, notification_type, event, user, node, timestamp,
     if notification_type == 'none':
         return
 
-    template = event + '.html.mako'
+    # Reviews has multiple email templates for the global_reviews event.
```

Comment:
Rather than a reviews-specific comment, explain how the arguments are used. Maybe something like "If \`template\` is not specified, default to using a template with name \`event\`"